package.json

```diff
@@ -105,8 +105,8 @@
     "react-helmet": "^5.2.1",
     "react-markdown": "^3.6.0",
     "react-modal": "^3.11.1",
-    "react-router": "^4.2.0",
-    "react-router-dom": "^4.2.2",
+    "react-router": "^5.1.2",
```

Comment:
The only breaking change (see v5.0.0 section) doesn't affect Pulse's [existing code](https://github.com/mozilla/network-pulse/blob/master/app.jsx#L3).